src/styles/blocks/card.scss

```diff
@@ -0,0 +1,120 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+
+  width: $card-width;
+  border-radius: 5px;
+
+  text-decoration: none;
+
+  border: 1px solid $border-card-color;
+  background: $background-card-color;
+  transition: transform $transition-duration, color $transition-duration;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+
+    margin-block: 32px 40px;
+  }
+
+  &__info {
+    padding-inline: 16px;
+    padding-bottom: 16px;
+  }
+
+  &__title {
+    display: block;
+    margin: 0;
+
+    margin-bottom: 4px;
+
+    color: $text-color-primary;
+    font-size: $text-size-m;
+    font-weight: 500;
+    line-height: 18px;
+    text-decoration: none;
+  }
+
+  &__product-code {
+    display: block;
+
+    margin-bottom: 16px;
+
+    color: $text-color-secondary;
+    font-size: $text-size-s;
+    line-height: 14px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: 24px;
```

Comment:
in situation like here you can delete the line.

actually try to use line only where you have big group of different styles;

check it other places as well